apps/todo/project.json

```diff
@@ -52,6 +52,15 @@
         }
       }
     },
+    "size-limit": {
```

Comment:
instead of doing it will be good to create a plugin that only runs on affected builds.